crowbar_framework/app/models/network_service.rb

```diff
@@ -413,8 +412,6 @@ def build_net_info(network, name, db = nil)
     db["network"].each { |k,v|
       net_info[k] = v unless v.nil?
     }
-    net_info["usage"]= network
-    net_info["node"] = name
```

Comment:
@aspiers pretty sure it's not needed here, as the info is already available somewhere else (which is why I removed it) :-)